diesel/src/sqlite/connection/raw.rs

```diff
@@ -204,7 +251,8 @@ extern "C" fn run_custom_function<F>(
 ) where
     F: FnMut(&RawConnection, &[*mut ffi::sqlite3_value]) -> QueryResult<SerializedValue>
         + Send
-        + 'static,
+        + 'static
+        + std::panic::RefUnwindSafe,
```

Comment:
As this is a `FnMut` and not a `Fn` I'm would guess that `RefUnwindSafe` is not the right trait here. We do potentially mutate `F` here.